source/features/index.tsx

```diff
@@ -44,6 +44,7 @@ interface InternalRunConfig {
 }
 
 let log: typeof console.log;
+export let logHTTP: typeof console.log;
```

Comment:
I tried for over an hour. I do not know how to do this any other way.

XO does not like this. Any suggestions?

Reply:
```suggestion
let logHTTP: typeof console.log;

export {logHTTP}
```

Or just silence the error.

Also it would be a good opportunity to use a default:
```ts
let log = console.log;
```

Reply:
> Or instead of exporting it here it should be exported on the `features` object, so it can be used as `features.logHTTP()`

It wont allow a `let` to be exported

Reply:
Better yet, we can have a single `log` object, like:

```ts
const log = {
	info: console.log,
	http: console.log,
	error: logError,
}
```

And then it can be overridden easily everywhere these methods are used. Could be a separate PR if you prefer.

Reply:
@yakov116 I have followed all fregante's suggestions. Is it ok if I push here?

Reply:
Yes

Reply:
Here. I tested it and it works ok.

Reply:
Where does it respect the user settings?

Reply:
If I click "Clear cache" and then ticking "Log API calls in the console" then logging becomes enabled.
Likewise, clicking "Clear cache" and unticking it, makes logging disabled.
This is how I tested it.

Do you mean something else? Have I got it wrong?

Reply:
What if you dont clear? (You should have to)

Reply:
You are right, thanks for noting.
Clearing cache is irrelevant. I had mistakenly removed two lines. 
I restored them and now it respects the setting again.